iot_verifier/src/rewarder.rs

```diff
@@ -1,11 +1,14 @@
 use crate::{
-    reward_share::{operational_rewards, GatewayShares},
+    reward_share::{self, GatewayShares},
     telemetry,
 };
 use chrono::{DateTime, Duration, TimeZone, Utc};
 use db_store::meta;
 use file_store::{file_sink, traits::TimestampEncode};
 use futures::future::LocalBoxFuture;
+use helium_proto::services::poc_lora as proto;
+use helium_proto::services::poc_lora::iot_reward_share::Reward as ProtoReward;
+use helium_proto::services::poc_lora::{UnallocatedReward, UnallocatedRewardType};
```

Comment:
```suggestion
use helium_proto::services::poc_lora as proto;
use helium_proto::services::poc_lora::iot_reward_share::Reward as ProtoReward;
use helium_proto::services::poc_lora::{UnallocatedReward, UnallocatedRewardType};
```

and then reference throughout as `proto::iot_reward_share::Reward`, `proto::UnallocatedReward`, `proto::UnallocatedRewardType`

Reply:
Addressed in sep PR via this commit: d1238dc8e5e255b4aa03f3c4a41fc6da62053b04